WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/BottomAnnouncementView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+
+struct BottomAnnouncementView: View {
+    @Environment(\.presentationMode) private var presentation
+
+    var buttonTapped: (() -> Void)?
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Spacer()
+            Text(Localization.title)
+                .font(.headline)
+                .padding(.bottom, Layout.titleFontSize)
+            Text(Localization.message)
+                .font(.caption)
+                .foregroundColor(.gray)
+                .padding(.bottom, Layout.messageFontSize)
+                .multilineTextAlignment(.leading)
+                .fixedSize(horizontal: false, vertical: true)
+            Button(Localization.confirmationButton) {
+                if let completionHandler = buttonTapped {
+                    completionHandler()
+                }
+                presentation.wrappedValue.dismiss()
+            }
+            .buttonStyle(SecondaryButtonStyle())
+            .padding(.bottom, Layout.bottomPadding)
+        }
+        .padding(.leading, Layout.horizontalPadding)
+        .padding(.trailing, Layout.horizontalPadding)
+        .padding(.top, Layout.verticalPadding)
+        .padding(.bottom, Layout.verticalPadding)
+        .background(RoundedRectangle(cornerRadius: Layout.cornerRadius)
+            .fill(Color(.withColorStudio(.gray, shade: .shade0))))
+    }
+}
+
+extension BottomAnnouncementView {
+    enum Localization {
+        static let title = NSLocalizedString("Payments from the Menu tab",
```

Comment:
If we create a view model we could pass these strings through the initializer, so this view can remain totally generic.

Reply:
Great idea, done here: 5a8abf6190078608d804c1c88dd9d4a50067ea18